.eslintrc

```diff
@@ -355,7 +356,7 @@
     // Require a space immediately following the // in a line comment.
     "spaced-comment": [2, "always"],
     // Require "use strict" to be defined globally in the script.
-    "strict": [2, "global"],
+    "strict": [0, "global"],
```

Comment:
This makes it so that we don't need "use strict"? How does the module system affect that?

We should probably keep "use strict"; at the top of all the files even if we could automatically do that in our build system, so that the code works the same way when we merge.

Sorry if I'm mis-interpreting, I'm not too familiar with eslint so it's a little hard to read all these tweaks.